### PR TITLE
Spring Cleaning

### PR DESCRIPTION
**Pull request by schweinsbergs**
added comments and removed functions that functionally did nothing (such as scraping())

### CODE REVIEW

**Comment by schweinsbergs** [comment]
Not quite sure how to actually add what I did this week. I got very little working, but since I'm working with a certain part of the dataframe, I'm going to insert what the ingredients list looks like for each recipe? 

 

This goes hand in hand with issue #19 , though does not solve it. It's what I'm working on. I've found that some options are the [fuzzy comparisons](https://towardsdatascience.com/fuzzy-string-matching-in-python-68f240d910fe), but I want to try out [regex](https://www.guru99.com/python-regular-expressions-complete-tutorial.html)

Feedback on that problem would be rad.

**Comment by jazzij** [comment]
You could definitely use regex on the ingredient list since the phrases are pretty consistent and regular. "## _measure_ ingredient (detail)" t
The details are inconsistent, some specify ingredient type while others specify quantity, so you might want to just drop those off the end.

File: Code/recipeApp.py [hunk 1]
```diff
@@ -118,25 +122,41 @@ def user_input(self):
...
         """
         Compares the dataframe's ingredients list and the user's.
         This might be helpful: https://towardsdatascience.com/dealing-with-list-values-in-pandas-dataframes-a177e534f173
+        https://stackoverflow.com/questions/5319922/python-check-if-word-is-in-a-string
+
+        https://www.guru99.com/python-regular-expressions-complete-tutorial.html
         :return:
         """
 
+        return re.compile(r'\b({0})\b'.format(w), flags=re.IGNORECASE).search
```
Comment (jazzij) [inline Code/recipeApp.py:135]: It looks like you're using regex to compare ingredient list, but that might be too hard? I think you could use regex to select the recipe ingredient out of each recipe phrase (i.e. strawberry out of 2 cups of strawberry), then use String comparison (recipe_ingredient in  user_ingredients) to see if the user has it

Comment (alfarozavalae) [reply Code/recipeApp.py:135]: ^^ I agree. It makes more sense to get rid of the other things in the recipe and just leave the ingredients for comparison.

**Comment by alfarozavalae** [comment]
I am assuming you created this pull request for your weekly code/documentation PR. Regarding that, on wednesdays you need to make a PR with code (your normal code, documentation, etc) to show what you have been working on. You do not necesarily need to add pictures, it can be just your normal code (if in Github) and add an issue telling us what you need us to review/give feedback on.

**Comment by jazzij** [comment]
This PR is ok to merge

**Pull request merged**